src/CheckVariables.ts

```diff
@@ -663,4 +664,11 @@ export const checkVariablesOnLoad = (data: PlayerSave) => {
     if (data.ascensionCounterRealReal === undefined) {
         player.ascensionCounterRealReal = 0;
     }
+
+    //Will re apply theme's on refresh
+    if (data.theme === undefined) {
```

Comment:
this shouldn't be saved on the player object at all

I would just use localStorage since it's not gonna change often

Reply:
Its only adding 1 number into data

Reply:
Dont want to use localStorage to allow imports to keep theme that you had

Reply:
Also its where its only on load or import is checked and loaded

Reply:
Also if you excepted me to move every color value into local storage, my answer is no